fury/actors/tests/test_effect_manager.py

```diff
@@ -0,0 +1,325 @@
+                   [0.08251756], [0.625351], [0.89982588],
+                   [0.62378987], [0.8661594], [0.05583442],
+                   [0.60157791], [0.84737657], [0.36433019],
+                   [0.13263502], [0.30937519], [0.88979053]])
+
+
+def test_window_to_texture(interactive=False):
+    fs_impl = """
+    vec2 res_factor = vec2(res.y/res.x, 1.0);
+    vec2 renorm_tex = res_factor*normalizedVertexMCVSOutput.xy*0.5 + 0.5;
+    vec4 tex = texture(screenTexture, renorm_tex);
+    fragOutput0 = vec4(tex);
+    """
+
+    on_scene = window.Scene()
+    on_scene.set_camera(position=(-2, 1, -2),
+                        focal_point=(0.0,
+                                     0.0,
+                                     0.0),
+                        view_up=(0.0, 0.0, 0.0))
+
+    on_manager = window.ShowManager(
+        on_scene,
+        "on_manager",
+        (width,
+         height))
+
+    off_scene = window.Scene()
+    off_scene.set_camera(position=(-4, 2, -4),
+                         focal_point=(0.0,
+                                      0.0,
+                                      0.0),
+                         view_up=(0.0, 0.0, 0.0))
+
+    off_manager = window.ShowManager(
+        off_scene,
+        "off_manager",
+        (width,
+         height))
+
+    off_manager.window.SetOffScreenRendering(True)
+    off_manager.initialize()
+
+    scale = np.array([[width/height, 1.0, 0.0]])
+    c = cube(np.array([[0.0, 0.0, 0.0]]), colors=(1.0, 1.0, 0.0))
+    bill = billboard(np.array([[0.0, 0.0, 0.0]]), scales=scale, fs_impl=fs_impl)
+    shader_custom_uniforms(bill, "fragment").SetUniform2f("res", off_manager.size)
+
+    off_manager.scene.add(c)
+    on_manager.scene.add(bill)
+
+    tex_name = "screenTexture"
+    blending_mode = "Interpolate"
+    dtype = "RGBA"
+    wrap_mode = "ClampToBorder"
+    border_color = (0.0, 0.0, 0.0, 1.0)
+    interpolate = True
+
+    off_manager.render()
+    window_to_texture(off_manager.window,
+                      tex_name,
+                      bill,
+                      blending_mode=blending_mode,
+                      wrap_mode=wrap_mode,
+                      border_color=border_color,
+                      interpolate=interpolate,
+                      d_type=dtype)
+
+    if interactive:
+        window.show(on_manager.scene)
+
+    n_textures = bill.GetProperty().GetNumberOfTextures()
+    texture = bill.GetProperty().GetTexture(tex_name)
+
+    npt.assert_equal(1, n_textures)
+    npt.assert_equal(WRAP_MODE_DIC[wrap_mode.lower()], texture.GetWrap())
+    npt.assert_equal(
+        BLENDING_MODE_DIC[blending_mode.lower()], texture.GetBlendingMode())
+    npt.assert_array_almost_equal(list(border_color), texture.GetBorderColor())
+    npt.assert_equal(interpolate, texture.GetInterpolate())
+    npt.assert_equal(True, texture.GetMipmap())
+
+    record(on_manager.scene, out_path="test_window.png", size=(width, height))
```

Comment:
Do you need this screen shot? As I see you are not testing anything on this it is better to remove it!